### ISSUE TITLE
A predict bug report.

### ISSUE THREAD

**Issue by dmkr001** [otenim/GLCIC-PyTorch#18]
Hi Sir,
Thanks for your code. But I find a small bug is that you forgot to use the model.train() and model.eval() to switch the performence of batch normalize layer. And when I train the dataset with different brightness, the predict result will become wired if I don't use the model.eval(). Hope it can help.

**Comment by otenim** [comment]
@dmkr001 

Fixed issues that you pointed out, thanks a lot :)
